.gitignore

```
@@ -27,3 +27,8 @@ Carthage/Build
 
 # CocoaPods
 Pods/
+
+# Swift Package Manager
+generated/
+.build/
+.swiftpm/
```

nit: Let's leave the newline at the end
```suggestion
.swiftpm/

```